tests/cephfs/cephfs_mirroring/test_cephfs_mirror_auth_caps.py

```diff
@@ -0,0 +1,523 @@
+import random
+import string
+import traceback
+
+from ceph.ceph import CommandFailed
+from tests.cephfs.cephfs_mirroring.cephfs_mirroring_utils import CephfsMirroringUtils
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    CEPH-83574109 - Modify the user capabilities with limited access to the mds and osd
+                    so that the snapshot creation should fail and observe the behavior on MDS
+    It has pre-requisites,
+    performs various operations, and cleans up the system after the tests.
```

Comment:
I feel this content is not necessary, as its implicit with all automation workflow.